crates/router/src/connector/authorizedotnet/transformers.rs

```diff
@@ -661,21 +729,22 @@ impl<F> TryFrom<&types::RefundsRouterData<F>> for CreateRefundRequest {
             })?
             .clone();
 
-        let merchant_authentication = AuthorizedotnetAuthType::try_from(&item.connector_auth_type)?;
+        let merchant_authentication =
+            AuthorizedotnetAuthType::try_from(&item.router_data.connector_auth_type)?;
 
         let transaction_request = RefundTransactionRequest {
             transaction_type: TransactionType::Refund,
             amount: utils::to_currency_base_unit_asf64(
-                item.request.refund_amount,
-                item.request.currency,
+                item.router_data.request.refund_amount,
```

Comment:
```suggestion
                amount : item.amount.clone(),
```

Reply:
amount in RefundTransactionRequest is a f64 and amount in AuthorizedotnetRouterData is a String